src/components/UserPortal/CreateDirectChat/CreateDirectChat.spec.tsx

```
@@ -297,33 +297,18 @@ describe('CreateDirectChatModal', () => {
-            email: 'user2@example.com',
-            firstName: 'Test',
-            lastName: 'User2',
-          },
-        ],
-        unseenMessagesByUsers: '',
         description: 'A direct chat conversation',
-        createdAt: dayjs.utc().toDate(),
+        createdAt: dayjs.utc().toISOString(),
+        members: {
+          edges: [
+            { node: { user: { id: '1', name: 'Current User' }, role: 'regular' } },
```

ESLint

[error] Replace `·node:·{·user:·{·id:·'1',·name:·'Current·User'·},·role:·'regular'·}` with `⏎··············node:·{⏎················user:·{·id:·'1',·name:·'Current·User'·},⏎················role:·'regular',⏎··············},⏎···········`

(prettier/prettier)